autorel/dist_tarball_builder/docker_cli.py

```diff
@@ -0,0 +1,79 @@
+"""
+    Module : docker
+    This module provides abstraction layer over docker-py
+"""
+import logging
+import sys
+from docker import Client
+
+
+class Docker(object):
```

Comment:
What if `Docker` would be able to receive an instance and then call a method on it _inside_ the docker container?